client/verta/verta/_cli/registry/get.py

```diff
@@ -40,6 +40,9 @@ def get_model(model_name, output, workspace):
     except ValueError:
         raise click.BadParameter("model {} not found".format(model_name))
 
+    if model is None:
+        raise click.BadParameter("model {} not found".format(model_name))
+
```

Comment:
I think this can be removed now, after https://github.com/VertaAI/modeldb/pull/1077 is merged

Reply:
removed